docs/index.md

```diff
@@ -66,6 +59,31 @@ can be used to:
     4.  Please also use the VM argument `-ea` (enable assertions) in your Preferences
         or in your Run/Debug configurations, so that failed assertions don’t just slip through.
 
+### Updating CLDR and ICU versions
+
+> :point_right: Note: This is a mess. See <https://unicode-org.atlassian.net/browse/ICU-21757>
+
+See the top level `pom.xml` under `<properties>`.
+
+#### ICU
+
+- Every time an ICU release/prerelease (not tag) is created on GitHub, a new package is created.
+- Go to <https://github.com/orgs/unicode-org/packages?repo_name=icu> to see what versions of ICU special packages are available.
+- Update `icu.version` to the version string, such as `70.0.1-SNAPSHOT-cldr-2021-09-15`
```

Comment:
optional: you can specify where `icu.version` should be set (the root `pom.xml`). It's mentioned in the parent heading. But a sibling heading below for "custom versions of CLDR" mentions this specifically.

Reply:
+1